android/src/main/java/tw/ktrssreader/config/KtRssReaderConfig.kt

```diff
@@ -28,5 +29,5 @@ class KtRssReaderConfig {
     var flushCache: Boolean = false
 
     @ExperimentalTime
-    var expiredTimeMillis: Long = 1.days.toLongMilliseconds()
+    var expiredTimeMillis: Long = Duration.days(1).inWholeMilliseconds
```

Comment:
its from here the initially reported issue comes from https://github.com/ivanisidrowu/KtRssReader/issues/69#issue-873652216